### ISSUE TITLE
++fats

### ISSUE THREAD

**Issue by IceCereal** [IceCereal/RobotSteve#2]
Long ago, one of our users wanted ++fats instead of ++stats. We recently figured out what it should do:

When a user summons ++fats, the bot generates a random profile about them as so:

```
IceCereal is 70% fat.
IceCereal is really feeling 20% lonely right now.
IceCereal eats 10.2kg of food every day.
IceCereal is indeed, 20% crap at Minecraft.
```

Create a new command in a new file in `utils/random` that responds to ++fats.

**Comment by AnantaSrikar** [comment]
I'd like to take this up

**Comment by IceCereal** [comment]
Sure. You can take it up.

**Comment by IceCereal** [comment]
Solved in #6 #7. Am closing.
